sdk/astronomer/Azure.ResourceManager.Astro/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (2024-05-01)
+
+### General New Features
```

Comment:
This must be one of the standard headers.   You cannot treat it as free-form text.